clouddriver-appengine/src/test/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/converters/DeployAppengineAtomicOperationConverterSpec.groovy

```diff
@@ -67,4 +68,25 @@ class DeployAppengineAtomicOperationConverterSpec extends Specification {
     then:
       operation instanceof DeployAppengineAtomicOperation
   }
+
+  void "input with an unsupported artifact type throws an AppengineDescriptionConversionException"() {
```

Comment:
Is there a case to be added to see a deploy with a container calls the correct stub here?